driver/src/main/java/org/neo4j/driver/internal/async/pool/ConnectionPoolImpl.java

```diff
@@ -60,10 +61,10 @@
     private final ConcurrentMap<BoltServerAddress,ChannelPool> pools = new ConcurrentHashMap<>();
     private final AtomicBoolean closed = new AtomicBoolean();
 
-    public ConnectionPoolImpl( ChannelConnector connector, Bootstrap bootstrap, PoolSettings settings,
+    public ConnectionPoolImpl( ChannelConnector connector, Bootstrap bootstrap, EventExecutorGroup eventExecutorGroup, PoolSettings settings,
             MetricsListener metricsListener, Logging logging, Clock clock )
     {
-        this( connector, bootstrap, new NettyChannelTracker( metricsListener, logging ), settings, metricsListener, logging, clock );
+        this( connector, bootstrap, new NettyChannelTracker( metricsListener, eventExecutorGroup, logging ), settings, metricsListener, logging, clock );
```

Comment:
I think `EventExecutorGroup` parameter can be removed. Channel tracker can be created using the given bootstrap:

```java
new NettyChannelTracker( metricsListener, bootstrap.config().group().next(), logging )
```